.changelog

```diff
@@ -4,6 +4,9 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Limit the changelog data in generated metadata in Administration Guide.
+  * The default number of entries is now 20 and it is consistent with
+  the number of entries from SUSE Linux Enterprise. (SUSE/spacewalk#19711)
```

Comment:
I'd remove the mention to the github issue, as we only add bsc, jira, CVEs.

Reply:
done